builder/eopkg.go

```diff
@@ -28,6 +28,12 @@ import (
 	"github.com/getsolus/libosdev/disk"
 )
 
+var (
+	installCommand   = "eopkg.bin"  // Command used for installing packages
+	ypkgBuildCommand = "ypkg-build" // Command used for building package.yml recipes
+	xmlBuildCommand  = "eopkg.py2"  // Command used for building pspec.xml recipes
```

Comment:
Might it be smart to leave some sort of searchable epoch note here?

Reply:
The real question is: will there still be pspec.xml files after the epoch? If not (which is my assumption), this variable will become unused (triggering a linter error) when the pspec xml code is removed.

Reply:
As discussed in the recent staff meeting, the 3rd party XMLs will technically be able to be ported to py3 it looks like.

So the answer to that question is a tentative "yes".

Reply:
Will put up a solbuild issue for us to reference in the eopkg migration epoch bump issue.